learn/advanced/filtering_and_faceted_search.md

```diff
@@ -102,10 +102,14 @@ The [`GET` route of the search endpoint](/reference/api/search.md#search-in-an-i
 
 String expressions combine conditions using the following filter operators and parentheses:
 
-- `NOT` only returns documents that do not satisfy a condition : `NOT genres = horror`
+- `NOT` returns all documents that do not satisfy a condition, including results missing the specified attribute. The expression `NOT genres = horror` returns all documents whose `genres` do not contain `horror` and all documents missing a `genres` field
```

Comment:
This is not strictly correct. To take a convoluted example:
```
NOT x NOT EXISTS
```
does not return "results missing the specified attribute". In fact, it only returns results with the specified attribute.

The example you chose with `NOT genres = horror` is very instructive though and I would keep it.

Reply:
hmmmm…yeah, I see what you mean. The example you gave does sound like a _very_ edgy edge case, but the problematic clause is not strictly necessary since the next sentence already describes what we need (that `NOT x = y` will return documents with `x = z` and documents without an `x` field). I'll remove it.